### DISCUSSION TITLE
Samsung Galaxy Smart Watch6 - how to install the Juggluco watch face?

### DISCUSSION THREAD

**Discussion by kdupke** [j-kaltes/Juggluco#171] [General]
Hi,

thank you for the WearOS version of Juggluco.

I am looking to enable the watch face for/from Juggluco on my Samsung Galaxy Smart Watch6.

My goal is to get my current glucose level and trend displayed on the watch itself, without the need to switch to the app. Either like displaying my oxygen level on an existing watch face or to replace the standard watch face by a specific one for Juggluco (which I understand is what to be meant by 'coming with a watch face'.

I installed Juggluco-8.0.24-small.apk yesterday via Wifi-ADB. Initialising the watch from the phone was quick and easy. So now I have Juggluco running on my watch. 

Unfortunately I am not able to find/configure the Juggluco watch face.

Long press on the watch gets my to the watch faces but I can't select nor add the Juggluco watch face.
Similar on the wearOS app on my phone.
In both cases I can select other watch faces, but not the Juggluco one.

Do I have to download/install something else on my watch?

best regards,
kai

**Comment by j-kaltes** [comment]
Why the hell did you install Juggluco-8.0.24-small.apk? Samsung Galaxy Watch6 runs WearOS and  https://www.juggluco.nl/Juggluco/download.html mentioned a WearOS version with Watch face and explicitly says that the version you installed doesn’t contain a watch face.

**Comment by kdupke** [comment]
Thank you for helping me and for your nice and friendly words.

Now I do know what mistake I made.

Not sure why I made the mistake, I must be just reading the pages the wrong way.

It might be good to mention the two versions on the watch page or to adjust the layout of the download page. But it might be simply that I was too stupid to get the difference. Now after you wrote it, it is pretty clear. Of course, it wasn't that clear to me before.

**Comment by kdupke** [comment]
I now installed Juggluco-8.0.24-wear-arm32.apk over the existing Juggluco-8.0.24-small.apk. When starting Juggluco its version shows the 'wear' version in the settings. 

I was able to find the watch face know in the wearOS app on my phone.

As you may know the details - any reason I can add pulse as complication but it does not show the measured rate but just has a button for me to measure the pulse? I may need to read though the info material as well.

**Comment by j-kaltes** [comment]
Try the heart rate complication in another non-Samsung watch face: it never shows the heart rate. For that reason the Juggluco watch face itself contains a heart rate. You can turn it on by long pressing the watch face, pressing on "Customize" and activating "Heart rate".

**Reply by kdupke** [reply]
Do I understand it right, that the hear rate is limited by design to the samsung watch faces, but you can trick around it?

However, when I select 'puls (German)' then I get a small heart displayed. I can tap the heart and it jumps to heart rate measurement. Just as in the samsung watch face. But it does  not show the heart pulse rate, as in the original watch face.

**Reply by j-kaltes** [reply]
I don't trick anything. The heart rate complication has this Samsung only restriction. Juggluco's watch face displays the heart rate without making use of that complication by using the WearOS interface for heart rate.

**Reply by kdupke** [reply]
Thank you for the explanation. And the work on this!
